features/step_definitions/external_work_steps.rb

```diff
@@ -20,14 +20,11 @@
   fill_in("bookmark_tag_string", with: DEFAULT_BOOKMARK_TAGS)
 end
 
-Given /^I bookmark the external work "([^\"]*)"$/ do |title|
-  visit new_external_work_path
-  fill_in("bookmark_external_url", with: DEFAULT_EXTERNAL_URL)
-  fill_in("bookmark_external_author", with: DEFAULT_EXTERNAL_CREATOR)
+Given /^I bookmark the external work "([^\"]*)"(?: with fandom "([^"]*)")?(?: with character "([^"]*)")?$/ do |title, fandom, character|
```

Comment:
This new step ends up unused after you switched to factories, should we keep it?